src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
I definitely agree we need a general solution to this problem. The test Bridge host will be driving the hostname used so long term it makes sense to have this problem solved via the config mechanism we use there. I had an itch I needed to scratch today which was get fiddler working on localhost for our existing outer loop tests so figured I would generalize the solution so others could easily benefit (and I wouldn't need to keep a local copy of these changes in sync :smiley: .) I have no problem with this being a stop-gap solution. I would like to see us not need to use fiddler in the long run as I would eventually like to have some inspection mechanism built into either the bridge or the client.